pkgs/applications/graphics/djview/default.nix

```diff
@@ -13,18 +13,47 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig ];
 
-  buildInputs = [ djvulibre qt4 xorg.libXt libtiff ]
+  buildInputs = [ djvulibre qtbase qttools xorg.libXt libtiff ]
```

Comment:
qttools should go into nativeBuildInputs.

Reply:
Thank you.
Done.